misc/anonymizer.go

```diff
@@ -0,0 +1,10 @@
+package misc
+
+import (
+	"context"
+	"io"
+)
+
+type Anonymizer interface {
```

Comment:
It is better to put the interfaces in a separate `interfaces` module.

Reply:
Done